backend/capellacollab/core/authentication/provider/oauth/keystore.py

```diff
@@ -14,6 +14,8 @@
 from capellacollab.config import config
 from capellacollab.core.authentication.provider import models
 
+from .. import models as provider_models
+
```

Comment:
I think we talked about that already and I forgot your answer but I thought relative imports make only sense in the same package which is why I am curious why did you decided to use it here :D 
[Google says no relative imports even though we are still using them](https://google.github.io/styleguide/pyguide.html#:~:text=Do%20not%20use%20relative%20names%20in%20imports.%20Even%20if%20the%20module%20is%20in%20the%20same%20package%2C%20use%20the%20full%20package%20name.%20This%20helps%20prevent%20unintentionally%20importing%20a%20package%20twice.)

Reply:
Yes, we discussed this topic in our planning meeting on the 12.06 and decided there to limit the scope of relative imports to up to one level above the current one. This is also currently written in our development documentation [code-style/imports](https://dsd-dbs.github.io/capella-collab-manager/code-style/#imports).
Personally, I at least would want to keep relative imports at the same level but I think we should then briefly discuss this topic in the next planning meeting.